docker-images/base-alpine-python36/harness.py

```diff
@@ -38,10 +39,10 @@ class ValidatorHarness:
     def __init__(self):
         self.version = self._find_version()
         self.validation_id = os.environ['AWS_BATCH_JOB_ID']
-        self.logger = get_logger('HARNESS')
+        self._parse_args()
+        self.logger = get_logger('HARNESS', "file_key", self.s3_object_key)
```

Comment:
I wonder if s3_object_key or validation_id is a better correlator.
I guess it depends what Ingest logs.

Reply:
@sampierson I chose the s3_object_key as it contains both the area_uuid and the filename which should allow for strong correlation with ingest. Agree/Disagree?

Reply:
Yes. That is fine.  I think the validation ID is later in the message anyway.  I hope that Ingest also logs the validation_id, so we can correlate multiple validation attempts of the same file.